src/main/webapp/app/core/main/controllers/productController.js

```diff
@@ -32,17 +30,18 @@ app.controller("ProductCtrl", function ($scope, $uibModal, $http, toastr,$locati
     };
 
     productCtrl.getProductPrice = function(product) {
-        var price = 0.0
-        if (product.price) {
+        var price = '';
+        if (product.statusCode == 1) {
             price = product.price;
+            var priceString = "R$" + price.toFixed(2); //com 2 casa decimais
         }
-        var priceString = price.toFixed(2); //com 2 casa decimais
+
         return priceString;
     }
 
     productCtrl.getProductStatus = function(product) { //Ver a lista da createProductDialogController
```

Comment:
what is this comment?

Reply:
Porque a função não fica clara oque faz por si só

Reply:
so, u have to refactor the function instead of add a comment. Add a comment in this situation is cleary a bad smell. Comments exists to explain why the code do something, and not how it does.